src/Runner.Discord/Responders/SteamGameResponder.cs

```diff
@@ -47,20 +47,20 @@ public SteamGameResponder(HttpClient httpClient)
         }
 
         public async Task ProcessMessage(IMessage message, CancellationToken token)
-        {
-            string randomGame;
-            if (message.Author.Id == 269883106792701952)
-            {
-                int totallyRandomAppId = RandomNumberGenerator.GetInt32(0, 1) == 0 ? 261820 : 582890;
-                randomGame = $"You should try this: {steamStoreUrl}{totallyRandomAppId}";
-                await message.Channel.SendMessageAsync(randomGame, options: token.ToRequestOptions());
-                return;
-            }
+        {            
             var trimmed = message.Content.ToLower().Trim();
             for (int i = 0; i < commands.Length; i++)
             {
                 if (trimmed.Contains(commands[i]))
                 {
+                    string randomGame;
+                    if (message.Author.Id == 269883106792701952)
+                    {
+                        int totallyRandomAppId = RandomNumberGenerator.GetInt32(0, 2) == 0 ? 261820 : 582890;
+                        randomGame = $"You should try this: {steamStoreUrl}{totallyRandomAppId}";
```

Comment:
This can just be declared where it's used, no need to delcare it outside of the `if`

Reply:
I'm reusing it at line 67